PowerEditor/src/Parameters.h

```diff
@@ -185,7 +185,7 @@ struct MapPosition
 struct sessionFileInfo : public Position
 {
 	sessionFileInfo(const TCHAR *fn, const TCHAR *ln, int encoding, bool userReadOnly, const Position& pos, const TCHAR *backupFilePath, FILETIME originalFileLastModifTimestamp, const MapPosition & mapPos) :
-		_isUserReadOnly(userReadOnly), _encoding(encoding), Position(pos), _originalFileLastModifTimestamp(originalFileLastModifTimestamp), _mapPos(mapPos)
+		Position(pos), _encoding(encoding), _isUserReadOnly(userReadOnly), _originalFileLastModifTimestamp(originalFileLastModifTimestamp), _mapPos(mapPos)

```

Comment:
Shouldn't it be `_encoding(encoding), _isUserReadOnly(userReadOnly), Position(pos)`?

Reply:
I tried to maximally respect order in constructor, but member initializers should be in same order as members appear/are declared. And since `Position` is declared earlier it must be "first".

Reply:
👍 
Thank you.